app/controllers/comments_controller.rb

```diff
@@ -1,10 +1,17 @@
 class CommentsController < ApplicationController
   before_action :set_comment, only: [:show, :edit, :update, :destroy]
+  # before_action :set_user
+  before_action :set_post
 
   # GET /comments
   # GET /comments.json
   def index
-    @comments = Comment.all
+    # if the current user exists, then
+    puts "Session user"
+    puts session[:user_id].class
+    if current_user
+      @comments = Comment.where(user_id: session[:user_id])
+    end
   end
```

Comment:
What happends if someone goes to this path while not logged in?

Reply:
They don't see any comments.